content/documentation/pages/6-recipes/7-scaling/0-index.md

```diff
@@ -0,0 +1,13 @@
+---
+path: 'recipes/scaling/'
+title: 'Scaling'
+description: 'Prometheus and Data Flow to autoscale streaming data pipelines'
+summary: true
+---
+
+# Scaling Data Pipelines
+
+Spring Cloud Data Flow provides a dedicated, [Scale API](%currentPath%/feature-guides/streams/scaling/), designed for scaling data pipelines. The API can be used to implement scaling characteristics based on application business logic. Developers can implement an auto-scale controller and reuse it with Kubernetes, Cloud Foundry or even with the Local platform for local testing purposes.
+
+Find below recipes about how to scale applications using SCDF Shell or implement autoscale streaming data pipelines.
```

Comment:
We are saying "find below", but then in the next line we are saying ..

> To find more about the scaling features provided with Spring Cloud Data Flow, visit the [stream scaling feature guide](%currentPath%/feature-guides/streams/scaling/).

Confused.